### DISCUSSION TITLE
[Solved] invalid mount config for type "bind"

### DISCUSSION THREAD

**Discussion by PaulaBras** [pterodactyl/panel#2982] [Support]
Hey, I have the problem, that I cant startup my servers, because of this error. I know that there is a permission error. I don't know were, I looked up the permissions on the Volumes themselves and tried to fix it. I also reinstalled multiple times Docker and or wings. Nothing helps. Sometimes they start for one run. That's it. For me, it's more random than a consistent error.

**Comment by trenutoo** [comment]
Are you running wings/panel inside the docker?

**Comment by PaulaBras** [comment]
no i dont. I close this because i found out that i used the wrong docker. If someone else used docker-ce it will not work after any restart. use docker.io
